### DISCUSSION TITLE
Visual Studio에서의 작업 - 1

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#14] [Ideas]
Visual Studio에서 작업 하는 방법을 깨우치고 있습니다.

![image](https://user-images.githubusercontent.com/62102830/191769002-e33a61f9-0044-4c1f-8ee6-af5b1a044069.png)
만약 새로운 cpp 파일이나 헤더를 만든다고 할때 경로를 Github문서로 해주시면 됩니다.

![image](https://user-images.githubusercontent.com/62102830/191769214-810af48e-e8ef-4416-803b-9aaa26602fd8.png)
![image](https://user-images.githubusercontent.com/62102830/191769233-795b33fb-5472-4cb7-a5e1-864ab5a9b235.png)
![image](https://user-images.githubusercontent.com/62102830/191769240-46517d68-7415-422e-a2f5-182508b426ee.png)

**Comment by Woon-2** [comment]
설마.. 지금까지는 어떻게 하셨나요?

**Reply by Lcat12345** [reply]
레퍼지토리 복사->기존 cpp 솔루션이 있는 폴더에 복사하려함 -> 당연히 안됨 -> 야발안해

**Reply by Lcat12345** [reply]
생각보다 수준이 응애입니다 선생님

**Reply by Woon-2** [reply]
`Git`은 `Local Repository`의 변화를 관찰하기 때문에 `Local Repository`가 곧 작업을 하는 폴더여야 효율적입니다.   
그것을 깨달으신 것 같네요.

레포지터리 폴더에 통째로 솔루션과 프로젝트를 두는 것도 좋습니다.

`.gitignore` 파일에 의해 `commit`을 해도 소스 파일들만 올라올 겁니다.

한번 실험해보시기 바랍니다.
